operator/build-operator.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -eu
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+cd "$script_dir"
+
+: "${VERSION:=latest}"
+
+api_version="console.lightbend.com/v1alpha1"
+docker_registry="lightbend-docker-registry.bintray.io"
+image_name="enterprise-suite/console-operator"
+full_docker_name="${docker_registry}/${image_name}:${VERSION}"
+
+echo "Building operator image ${full_docker_name}..."
+
+if [[ "$VERSION" == "latest" ]]; then
+    echo "Using local helm chart..."
+    new_operator_args=--helm-chart="$script_dir"/../enterprise-suite
+else
+    echo "Using published ${CONSOLE_TAG} helm chart..."
+    new_operator_args=--helm-chart-repo=https://repo.lightbend.com/helm-charts \
+        --helm-chart=enterprise-suite --helm-chart-version="${CONSOLE_TAG}"
+fi
+
+rm -rf build && mkdir build
+
+cd build
+operator-sdk new console-operator --type=helm --kind=Console --api-version=${api_version} \
+    ${new_operator_args}
+
+cd console-operator
+operator-sdk build "${full_docker_name}"
```

Comment:
Do you worry that someone might ever accidentally create a `build` directory for other reasons and get a nasty surprise?  Just wondering if it might be an idea to use a more unusual name here.  Or not.